tests/system/action/organization/test_initial_import.py

```diff
@@ -0,0 +1,531 @@
+                        "privacy_policy": "",
+                        "login_text": "Welcome to OpenSlides. Please login.",
+                        "reset_password_verbose_errors": False,
+                        "enable_electronic_voting": False,
+                        "limit_of_meetings": 0,
+                        "committee_ids": [1],
+                        "active_meeting_ids": [1],
+                        "resource_ids": [],
+                        "organization_tag_ids": [],
+                        "theme_id": 1,
+                        "theme_ids": [1],
+                    }
+                },
+                "theme": {
+                    "1": {
+                        "id": 1,
+                        "name": "OpenSlides Blue",
+                        "accent_500": "#2196f3",
+                        "primary_500": "#317796",
+                        "warn_500": "#f06400",
+                        "organization_id": 1,
+                        "theme_for_organization_id": 1,
+                        "accent_100": None,
+                        "accent_200": None,
+                        "accent_300": None,
+                        "accent_400": None,
+                        "accent_50": None,
+                        "accent_600": None,
+                        "accent_700": None,
+                        "accent_800": None,
+                        "accent_900": None,
+                        "accent_a100": None,
+                        "accent_a200": None,
+                        "accent_a400": None,
+                        "accent_a700": None,
+                        "primary_100": None,
+                        "primary_200": None,
+                        "primary_300": None,
+                        "primary_400": None,
+                        "primary_50": None,
+                        "primary_600": None,
+                        "primary_700": None,
+                        "primary_800": None,
+                        "primary_900": None,
+                        "primary_a100": None,
+                        "primary_a200": None,
+                        "primary_a400": None,
+                        "primary_a700": None,
+                        "warn_100": None,
+                        "warn_200": None,
+                        "warn_300": None,
+                        "warn_400": None,
+                        "warn_50": None,
+                        "warn_600": None,
+                        "warn_700": None,
+                        "warn_800": None,
+                        "warn_900": None,
+                        "warn_a100": None,
+                        "warn_a200": None,
+                        "warn_a400": None,
+                        "warn_a700": None,
```

Comment:
I think all not-required fields should be left out. This is just a huge verbose block which contributes nothing to the actual tests. You probably have to adjust the checker for this.

Reply:
same for all other empty (`None`/`[]`) fields

Reply:
Removed.

Reply:
As I wrote above, you'll have to adjust the checker for this to work. I don't know why we implemented it this way in the first place, at least I don't know any cases where it would lead to unwanted behaviour to leave optional fields out.